rollup.config.js

```diff
@@ -48,6 +150,9 @@ const buildPlugins = ({ variant, minify, babelPlugins }) => {
       configFile: path.resolve(dirname, "babel.config.cjs"),
       plugins: babelPlugins,
     }),
+    bundleSizePlugin({
+      output: "bundlesize.json",
```

Comment:
If you run the custom build command from a folder outside the alloy directory, you will get the `bundlesize.json` file saved along with the alloy library file.

I think we should either not run this plugin when the custom build command is executed or change the path and use the dirname variable that is available in the file.